sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -97,11 +98,11 @@
      * @throws ResourceModifiedException when invalid certificate policy configuration is provided.
      * @return A {@link Poller} polling on the create certificate operation status.
      */
-    public Poller<CertificateOperation> createCertificate(String name, CertificatePolicy policy, Map<String, String> tags) {
-        return new Poller<CertificateOperation>(Duration.ofSeconds(1), createPollOperation(name), activationOperation(name, policy, tags), cancelOperation(name));
+    public Poller<CertificateOperation, Certificate> beginCreateCertificate(String name, CertificatePolicy policy, Map<String, String> tags) {
+        return new Poller<>(Duration.ofSeconds(1), createPollOperation(name), activationOperation(name, policy, tags), fetcResultOperation(name), cancelOperation(name));
     }
```

Comment:
Missing an `h` in the method name, fetc**h**ResultOperation?

Reply:
fixed